modules/services/davmail.nix

```diff
@@ -0,0 +1,142 @@
+{ config, lib, pkgs, ... }:
+let
+
+  inherit (lib)
+    attrNames boolToString concatMap concatStringsSep isAttrs isBool
+    mapAttrsRecursive mkDefault mkEnableOption mkIf mkOption optionalAttrs
+    types;
+
+  cfg = config.services.davmail;
+
+  settingsType = with types;
+    oneOf [ (attrsOf settingsType) str int bool ] // {
+      description =
+        "DavMail settings type (str, int, bool or attribute set thereof)";
+    };
+
+  toStr = val: if isBool val then boolToString val else toString val;
+
+  linesForAttrs = attrs:
+    concatMap (name:
+      let value = attrs.${name};
+      in if isAttrs value then
+        map (line: name + "." + line) (linesForAttrs value)
+      else
+        [ "${name}=${toStr value}" ]) (attrNames attrs);
+
+  settingsFile = pkgs.writeText "davmail.properties"
+    (concatStringsSep "\n" (linesForAttrs cfg.settings));
+
+in {
+
+  meta.maintainers = [ lib.maintainers.bmrips ];
```

Comment:
```suggestion
  meta.maintainers = [ lib.hm.maintainers.bmrips ];
```

Reply:
Ah, that's why the build fails. Thank you!